src/controller/python/chip/interaction_model/Delegate.h

```diff
@@ -23,8 +23,27 @@
 
 namespace chip {
 namespace python {
+
 static constexpr ClusterStatus kUndefinedClusterStatus = 0xFF;
-}
+
+// This needs to match the python definition that uses the same name.
+struct PyInvokeRequestData
+{
+    void * mCommandPath;
```

Comment:
should we specify the type of data this points to? void* is very generic. Same question for mAttributePath below.

Reply:
It is no longer a pointer and just a struct